plugins/Events/Reports/Base.php

```diff
@@ -67,9 +68,11 @@ protected function configureFooterMessage(ViewDataTable $view)
             return;
         }
 
-        $out = '';
-        EventDispatcher::getInstance()->postEvent('Template.afterEventsReport', array(&$out));
-        $view->config->show_footer_message = $out;
+        $view->config->filters[] = function(DataTable $dataTable) use ($view) {
+            $out = '';
+            EventDispatcher::getInstance()->postEvent('Template.afterEventsReport', [&$out, $dataTable]);
+            $view->config->show_footer_message = $out;
```

Comment:
Note: Moving this to a filter was required, as we otherwise aren't able to access the datatable at this point.